hawkbit-device-simulator/src/main/java/org/eclipse/hawkbit/simulator/http/NoAuthSecurityConfiguration.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2019 Bosch Software Innovations GmbH and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.hawkbit.simulator.http;
+
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+
+/**
+ * Spring security configuration.
+ * Depending on the ~.auth.enabled property access is granted for all requests.
+ */
+@EnableWebSecurity
```

Comment:
check if we really need this annotation here, and then probably add it to BasicAuthSecurityConfiguration class